src/Manager/UserManager.php

```diff
@@ -35,7 +35,7 @@ public function manageCreateUser(User $user, ?string $password = null): void
     /**
      * {@inheritdoc}
      */
-    public function manageUpdateUser(User $user, ?string $password = null): void
+    public function manageUpdateUser(User $user, bool $persist = true, ?string $password = null): void
```

Comment:
Mais non ! pourquoi tu as remis ! Il faut juste bien lire l'erreur qui s'affiche... :( ca fonctionnait quand on l'a fait ensemble non ?

Reply:
Done,

lorsque le fait un : vendor/bin/phpunit --coverage-html web/test-coverage

j'ai en retour : 

Remaining self deprecation notices (1)

Remaining indirect deprecation notices (2)
  1x: The "App\Manager\UserManager::manageUpdateUser()" method will require a new "bool $persist" argument in the next major version of its interface "App\Manager\UserManagerInterface", not defining it is deprecated.
    1x in DeprecationErrorHandler::shutdown from Symfony\Bridge\PhpUnit.

Parce que j'ai oublié de retirer : 
     * @param bool        $persist  save the User if true
Problème résolut.